helm/app-operator/templates/_resource.tpl

```diff
@@ -28,14 +28,14 @@ room for such suffix.
 {{- end -}}
 
 {{/*
-The unique deployment of app-operator is for management cluster app CRs and uses
-a special app version of 0.0.0.
+The deployment of app-operator for management cluster reconciles a special app
+CR version of 0.0.0.
 */}}
 {{- define "resource.app.unique" -}}
-{{- if hasSuffix "-unique" .Release.Name }}true{{ else }}false{{ end }}
+{{- if eq $.Chart.Name $.Release.Name }}true{{ else }}false{{ end }}
```

Comment:
I think integration testing is failing since the app-operator doesn't reconcile with app CRs having a `0.0.0` label. 
https://github.com/giantswarm/app-operator/blob/3e9d6cbdcab201b70d17735107c0b0734fc948ab/integration/test/watcher/configmap/configmap_test.go#L120-L127
Are the chart name and the release name is same on our integration test?

Reply:
Ah possibly we also need to change app-operator app name to drop the -unique suffix.

Reply:
@kopiczko Let me fix it on this branch, let's see it pass the integration test.